deploy/k8s/gprofiler.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           resources:
             requests:
               cpu: 100m
-              memory: 256m
+              memory: 256Mi
```

Comment:
Same for `k8s/helm-charts/values.yaml`

Reply:
Oh I actually checked another helm file and missed it.

Anyway it's retarded that this is cloned. Will fix in another PR